### ISSUE TITLE
How can yolov8 perform label polygon processing for object detection ?

### ISSUE THREAD

**Issue by trakhanh** [ultralytics/ultralytics#14752]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I have a question why I can train with polygons labels when I labels on roboflow and export it train it with yolov8_n->x as well. What makes it possible to identify and draw the bbox because when we do the classic style it only has 4 xywh points to detect? How can it read the polygon to perform object detection? Thank you if anyone can answer this for me

**Comment by Y-T-G** [comment]
You can easily get the bbox from the polygons.

**Comment by trakhanh** [comment]
Yes I know that, I mean I could include the polygon label to make it work. I'm just wondering why it's possible. I read an article about this but it's not very clear: https://blog.roboflow.com/polygons-object-detection/#how-to-use-polygons-with-object-detection-models

**Comment by Y-T-G** [comment]
ultralytics converts the polygon labels to bbox internally before training and uses that. It's not training on the polygon labels.

**Comment by trakhanh** [comment]
Thank you for answering my questions

**Comment by pderrenger** [comment]
You're welcome! If you have any more questions or need further clarification, feel free to ask. We're here to help!
